dev-haskell/glambda/glambda-1.0.2-r1.ebuild

```
@@ -0,0 +1,37 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.9.0.0.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="A simply typed lambda calculus interpreter, written with GADTs"
+HOMEPAGE="https://github.com/goldfirere/glambda"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+SRC_URI="${SRC_URI} https://github.com/goldfirere/glambda/pull/11.patch"
```

Could you give this a less generic name than `11.patch`?

```suggestion
SRC_URI+="
	https://github.com/goldfirere/glambda/pull/11.patch
		-> ${P}-prettyprinter.patch
"